roles/system-base/files/opt/ivy/install_datadog.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
 
+if [ -z "${DD_API_KEY+x}" ]; then
+  DD_API_KEY='ivy-use-set-datadog-key'
```

Comment:
can we set this to something more obvious like `KEY_UNSET` - or can we refactor this script to become an Ansible task that installs the agent and configures the `bind_host`, then we can use `set_datadog_key` at instance boot to enable it so we don't need to set a key at all?

Reply:
how about now?